README.md

```diff
@@ -5,11 +5,12 @@
 [![Build Status](https://circleci.com/gh/windmilleng/tilt/tree/master.svg?style=shield)](https://circleci.com/gh/windmilleng/tilt)
 [![GoDoc](https://godoc.org/github.com/windmilleng/tilt?status.svg)](https://godoc.org/github.com/windmilleng/tilt)
 
-Local Kubernetes development with no stress.
+Kubernetes for Prod, Tilt for Dev
 
-[Tilt](https://tilt.dev) helps you develop your microservices locally.
-Run `tilt up` to start working on your services in a complete dev environment
-configured for your team.
+Modern apps are made of multiple services: a database, an API server, a web bundler.
+[Tilt](https://tilt.dev) powers multi-service development.
+Run `tilt up` to start working in a complete dev environment configured
+for your team.
 
 Tilt watches your files for edits, automatically builds your container images,
 and applies any changes to bring your environment
```

Comment:
The way this is phrased makes it seem like three separate points, not like one continuous operation. Rephrase?

Reply:
ya, i agree. borrowed some of the language from the example guides, how's this?

> Tilt automates all the steps from a code change to a new process: watching files, building container images, and bringing your environment up-to-date.

Reply:
Perfect!